client/src/components/material.jsx

```diff
@@ -139,12 +138,25 @@ class Material extends React.Component {
 
   render() {
     const materials = this.props.materials;
+    if(!materials.length) {
+      return <Carousel>
+          <Card key='none' onMouseEnter={this.handleHover} onMouseLeave={this.handleHover} style={{backgroundColor: "#f5d90a"}}>
+          {this.state.isHovered ? <MatInfo style={{color: "black", fontSize: "2rem"}}>Let’s quack this case 🐣.</MatInfo> : null }
+          <Blob url="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT2EGNOn0R__P2Bv_oIzb2EIRAQCAYKO7xQaw&usqp=CAU"></Blob>
+          <MatName>100% vegan, range free rubber duck</MatName>
+          <MatDesc>What did the duck detective say to his partner?</MatDesc>
+          <Program>Program</Program>
+          <LearnMore>Learn More</LearnMore>
```

Comment:
if only we could click on learn more and it would take us to more info about quacking the case!

Reply:
lol! that's out of scope